app-creator/client/src/widgets/tool-bar/tool-bar.ts

```diff
@@ -77,20 +80,39 @@ export class ToolBar extends LitElement {
       margin-right: var(--tight);
     }
 
+    #import-textarea {
+      width: calc(100% - 2 * var(--tight) - 44px);
+      height: 250px;
+      margin-left: 24px;
+      padding: var(--tight);
+    }
+
+    #import-button {
+      background-color: var(--primary-color);
+      color: var(--accent-color);
+      height: 30px;
+      font-size: 0.8rem;
+      border: 0.3px solid var(--accent-color);
+    }
+
     #export-button {
       background-color: var(--accent-color);
       color: var(--primary-color);
       height: 30px;
       font-size: 0.8rem;
     }
+
+    #invalid-json-toast {
+      background-color: var(--validation-error-red-color);
+    }
   `;
 
   /**
    * Triggered when export button is clicked. It displays the paper dialog which
    * contains the serialized template string.
    */
-  openDialog() {
-    const dialog = this.shadowRoot?.querySelector('paper-dialog');
+  openExportDialog() {
+    const dialog = this.shadowRoot?.querySelector('#export-dialog');
```

Comment:
Could this be replaced with the `@query` decorator? Then we could just use `this.exportDialog` to reference the element. See https://lit-element.polymer-project.org/api/modules/_lit_element_.html#query